site/en/docs/devtools/workspaces/index.md

```diff
@@ -229,7 +229,7 @@ to your site. There's a way to have the **Sources** panel open alongside other p
 Congratulations! You now know how to save to sources the changes you make in DevTools to your locally hosted website.
 {% endAside %}
 
-You can set up multiple folders as **Workspaces**. All such folders are listed in {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [**Settings** > **Workspace**](/docs/devtools/settings/#workspace).
+You can set up multiple folders in a **Workspace**. All such folders are listed in {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [**Settings** > **Workspace**](/docs/devtools/settings/#workspace).
```

Comment:
capital or lowercase? There's only one workspace in DevTools. should it be the workspace?